docs/commands/ft.create.md

```diff
@@ -85,7 +85,7 @@ When dropped, a temporary index does not delete the hashes as they may have been
   memory but does not allow sorting based on the frequencies of a given term within
   the document.
 
-* **STOPWORDS**: If set, we set the index with a custom stopword list, to be ignored during
+* **STOPWORDS {num}**: If set, we set the index with a custom stopword list, to be ignored during
```

Comment:
In the command syntax at the top of the (generated) page, which is according to the [commands.json](https://github.com/RediSearch/RediSearch/blob/7f3317fc675fdf3f3f63acd8dfe457304e133782/commands.json#L129) file, it is named `count`, so seems clearer to use it also here
```[STOPWORDS count stopword [stopword ...]]```

Reply:
Makes sense